### ISSUE TITLE
Mp3 playback issue on Windows

### ISSUE THREAD

**Issue by chchwy** [pencil2d/pencil#1237]
### Issue Summary

Pencil2D do not playback certain mp3s properly on Windows. Currently, It's reported on Windows 7. Haven't tested it yet on Windows 10.

### Possible solutions

Build Pencil2D with Qt 5.13 and run it with parameters as below:
```
QT_MULTIMEDIA_PREFERRED_PLUGINS=w,d 
QT_MULTIMEDIA_PREFERRED_PLUGINS=gst,w,d. 
```
### System Information
+ Pencil2D Version: v0.6.4

**Comment by scribblemaniac** [comment]
When this issue occurs, sound files will appear as a single frame in duration like this:

![screenshot](https://user-images.githubusercontent.com/3461051/61840216-b0985980-ae4d-11e9-98ac-cb135f2c1798.png)

This has been confirmed not to affect macOS. There is a small chance it can affect Linux too as this has not been tested, but if you are have problems with sound on linux, it is likely the result of issue #1033, which is separate from this issue.

This is a playback issue and not an importing issue as importing a broken mp3 on windows and saving and opening that project file on a mac allows the sound files to be played properly.

**Comment by chchwy** [comment]
A piece of bad news is all 4 mp3 audios work on my Windows 10 machine.
Tested both on Qt 5.9.8 & Qt 5.13.0 without any `QT_MULTIMEDIA_PREFERRED_PLUGINS` settings.

**Comment by chchwy** [comment]
with `QT_MULTIMEDIA_PREFERRED_PLUGINS=w,d`, All mp3s work well

**Comment by chchwy** [comment]
with `QT_MULTIMEDIA_PREFERRED_PLUGINS=d`, All mp3s work well

**Comment by scribblemaniac** [comment]
None of the 4 test mp3 audio files work on my Windows 10 machine with. Tested with the v0.6.4-hotfix release. It's possible that we have different audio plugins/codecs installed

**Comment by scribblemaniac** [comment]
Closing this as it was addressed in #1258 by converting mp3 files to wavs automatically when importing.
